Modules/_ctypes/_ctypes.c

```diff
@@ -5747,6 +5747,22 @@ wstring_at(const wchar_t *ptr, int size)
     return PyUnicode_FromWideChar(ptr, ssize);
 }
 
+static PyObject *
+memoryview_at(void *ptr, Py_ssize_t size, int readonly)
+{
+    if (PySys_Audit("ctypes.memoryview_at", "nni",
+                    (Py_ssize_t)ptr, size, readonly) < 0) {
```

Comment:
Is there a possibility of data loss with this cast if `sizeof(Py_ssize_t) != sizeof(intptr_t)`?

Reply:
For the audit? Yes. `n` is the closest `PySys_Audit`/`PyBuildValue` have to `intptr_t`, though. 
There's a bunch of these across ctypes. If we want to change it, should go in a separate PR.